icicle/appUtils/ntt/ntt.cu

```diff
@@ -686,9 +704,8 @@ namespace ntt {
   }
 
   template <typename S>
-  NTTConfig<S> DefaultNTTConfig()
+  NTTConfig<S> DefaultNTTConfig(const device_context::DeviceContext& ctx)
```

Comment:
Totally agree. It's a workaround to the static stream bug but once Vitaly's PR is merged I can remove this.
Alternatively can add to MSM. What do you think?

Reply:
ok, let's wait for Vitaly's PR then

Reply:
After looking into this some more, it seems that avoiding this it would be breaking rust code (can explain why) so I added it to msm config too for symmetry. Note that the default value is default device_context like it is today